api/cluster.capnp

```diff
@@ -6,6 +6,25 @@ $Go.package("cluster");
 $Go.import("github.com/wetware/ww/internal/api/cluster");
 
 
+interface Anchor {
+    ls @0 (path :List(Text)) -> (children :List(Child));
+    walk @1 (path :List(Text)) -> (anchor :Anchor);
+
+    struct Child {
+        name @0 :Text;
+        anchor @1 :Anchor;
+    }
+}
+
+interface Host extends(Anchor) {
+    host @0 () -> (host :Text);
```

Comment:
I don't think this should be an RPC method. We already have this information in `HostAnchor`.  Users can access it by doing doing `a.(HostAnchor).Peer`.